_includes/tutorials/discount-promo/confluent/code/tutorial-steps/dev/source.json

```diff
@@ -25,9 +25,9 @@
   "connection.user"          : "postgres",
   "connection.password"      : "<my-database-password>",
   "db.name"                  : "<db-name>",
-  "table.whitelist"          : "orders",
+  "table.whitelist"          : "order_stream",
```

Comment:
Nit: Raising the possibility of having a table with the name `stream` could be confusing. FYC name this something very generic like `order_data` or similar

Reply:
+1

Reply:
Agree and fixed in latest commit